backend/backend/routing/matching/__init__.py

```diff
@@ -43,7 +43,8 @@ def get_matches(route: LineString, matchers: List[RouteMatcher]) -> Iterable[LSA
     """
     Return all LSA's that match the route.
     """
-    matched_lsas = LSA.objects.all()
+    # Only consider LSA's that are also for cyclists
+    matched_lsas = LSA.objects.filter(lsametadata__lane_type__icontains="Radfahrer")
```

Comment:
How much overhead does this produce?
Maybe should add a DB index for lane_type.

Reply:
Going to measure it.

Reply:
With the filter:
```
0.0016646385192871094 seconds
```
and
```
0.001116037368774414 seconds
```

With the old variant where we just got all LSAs:
```
0.00024771690368652344 seconds
```
and
```
0.0001232624053955078 seconds
```